<launch>
  <node name="interaction_generator" pkg="cr_week6_test" type="interaction_generator.py"/>
  <node name="perception_filter" pkg="cr_week6_test" type="perception_filter.py"/>
  <node name="robot_controller" pkg="cr_week6_test" type="robot_controller.py"/>
  <node name="robot_expression_prediction" pkg="cr_week6_test" type="robot_expression_prediction.py"/>

  
</launch>









